### PR TITLE
[Improvement] Sidebar footer

### PR DESCRIPTION
**Pull request by raczyk**

### CODE REVIEW

File: src/components/sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -128,6 +151,7 @@ export const Sidebar: FunctionComponent<ISidebarProps> = ({
   title,
   side,
   isOpen,
+  footer,
```
Comment (khola) [inline src/components/sidebar/Sidebar.tsx:154]:
```suggestion
  footer: Footer,
```

File: src/components/sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -181,6 +204,7 @@ export const Sidebar: FunctionComponent<ISidebarProps> = ({
                   </Col>
                 </Row>
               </Container>
+              {footer && <FooterSidebar>{footer()}</FooterSidebar>}
```
Comment (khola) [inline src/components/sidebar/Sidebar.tsx:207]:
```suggestion
              {Footer && <FooterSidebar><Footer /></FooterSidebar>}
```

File: src/components/sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -5,24 +5,28 @@ import { space } from "styled-system";
...
+  position: sticky;
+  top: 0;
+  right: 0;
+  left: 0;
   display: flex;
   width: 100%;
   flex-wrap: nowrap;
   align-items: center;
   border-bottom: ${theme.borders.grey};
+  background-color: ${theme.colors.white};
```
Comment (Horay) [inline src/components/sidebar/Sidebar.tsx:27]: it's better to use snowWhite color

File: src/components/sidebar/Sidebar.tsx [hunk 4]
```diff
@@ -85,13 +89,17 @@ const CloseLayer: any = styled.div<ICloseLayer>`
...
   overflow-x: hidden;
   overflow-y: scroll;
   ${bp("xs")`width: 100vw`};
   ${bp("sm")`width: 50vw`};
   ${bp("md")`width: 45vw`};
   ${bp("lg")`width: 40vw`};
+  ${bp("xl")`width: 30rem`};
+  ${bp("xxl")`width: 30rem`};
   box-shadow: ${theme.shadow.shadow04};
   background-color: ${theme.colors.white};
```
Comment (Horay) [inline src/components/sidebar/Sidebar.tsx:104]: it's better to use snowWhite color

File: src/components/sidebar/Sidebar.tsx [hunk 5]
```diff
@@ -103,12 +111,28 @@ const Inner: any = styled.div<IInnerProps>`
...
   z-index: ${theme.zindex.topoftheworld};
 `;
 
+const FooterSidebar = styled.div`
+  position: sticky;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  display: flex;
+  background-color: ${theme.colors.white};
```
Comment (Horay) [inline src/components/sidebar/Sidebar.tsx:129]: it's better to use snowWhite color

**Review by Horay — APPROVED** [review]

**Review by khola — APPROVED** [review]

**Pull request merged**